### DISCUSSION TITLE
issue rendering {{ .value.test }}

### DISCUSSION THREAD

**Discussion by sabreensalama** [mikefarah/yq#1760] [Q&A]
Hi, I need to add new field to template file using yq4 but everytime, I get a strange result, here is the commnd
`yq4 eval --inplace '.spec.template.spec +=  {"subdomain": "{{.Values.global.deployment.subdomain}}"}'  test.yml
`
I got it with the below format
`      subdomain: '{{`{{`}}.Values.global.deployment.subdomain{{`}}`}}'
`
but I need it to be:
`      subdomain:  {{ .Values.global.deployment.subdomain }}
`
is there any help?

**Comment by mikefarah** [comment]
It looks like you're trying to set it as a HELM template - which isn't actually valid yaml (the issue is the `{{` and `}}`). Yaml is a super set of JSON, so it when it sees `{` it thinks you're specifying an object, and it's not valid to have another `{` after it.
